media/js/facebookapps/App.js

```diff
@@ -101,8 +101,13 @@ DOWNLOADTAB.classes.App = (function (singleton) {
                 self.theater.showScene(2);
             }
 
-            if (self.window._gaq) {
-                self.window._gaq.push(['_trackPageview', self.virtualUrl]);
+            if (self.window.gaTrack) {
```

Comment:
Good news is that we should be good to modify this file.

Do we need the `if` statement above? Is it there just to make sure `global.js` has loaded?